docker_run.sh

```diff
@@ -13,15 +13,14 @@ npm rebuild node-sass
 php artisan lang:js --no-lib resources/js/translations.js
 chmod -R 777 public
 
-npm run dev&
 php artisan key:generate
 php artisan cache:clear
 php artisan config:clear
 
 # Ensure we have the admin user
 echo "User::create(['name'=>'Jane Bloggs','email'=>'jane@bloggs.net','password'=>Hash::make('passw0rd'),'role'=>2,'consent_past_data'=>'2021-01-01','consent_future_data'=>'2021-01-01','consent_gdpr'=>'2021-01-01']);" | php artisan tinker
 
-php artisan serve --host=0.0.0.0 --port=80
+php artisan dev --no-logs
```

Comment:
We might consider naming this file in a way that it expresses it's a dev run? Peeking the file and it's name, id otherwise expect this to be the production entrypoint?

Not a blocker

Reply:
Are you referring to `docker_run.sh`? If so, I agree and have some ideas already on how to differentiate between this dev setup and what would become the production setup for the docker image.